Client/mods/deathmatch/logic/CUnoccupiedVehicleSync.cpp

```diff
@@ -250,6 +260,10 @@ void CUnoccupiedVehicleSync::UpdateStates()
             g_pNet->DeallocateNetBitStream(pBitStream);
         }
     }
+
+    // Remove our vehicle again
+    if (pVehicle)
+        m_List.remove(pVehicle);
 }
 
```

Comment:
The reason I put this here here (vs. sending it from `CNetAPI::DoPulse`), is that it must send the packets at the correct rate as remote clients expect them to come in at 400 ms or whatever the sync rate is and apply interpolation based on that.
Maybe there's a better way to achieve this.